src/style.css

```diff
@@ -0,0 +1,37 @@
+.body {
+  margin: 0;
+  display: flex;
+  flex-direction: column;
+  height: 96px;
+}
+
+.stars {
+  display: flex;
+}
+
+.stars_star {
+  height: 16px;
+  width: 16px;
+  background-repeat: no-repeat;
+  background-position: center;
+}
+
+.stars_star:not(:first-child) {
+  margin-left: 4px;
+}
+
+.stars--0 .stars_star,
+.stars--1 .stars_star:not(:first-child),
+.stars--2 .stars_star:not(:nth-child(-n+2)),
+.stars--3 .stars_star:not(:nth-child(-n+3)),
+.stars--4 .stars_star:not(:nth-child(-n+4)) {
+  background-image: url(images/star.svg);
+}
```

Comment:
No need to use modifiers for default non-active image, you can set it to `stars_star` class